src/routes/jira/jira-sync-post.ts

```diff
@@ -19,7 +22,13 @@ export const JiraSyncPost = async (req: Request, res: Response, next: NextFuncti
 			res.status(404).send("Subscription not found, cannot resync.");
 			return;
 		}
-		await findOrStartSync(subscription, req.log, syncType);
+
+		if (commitsFromDate && commitsFromDate.valueOf() > Date.now()){
```

Comment:
Since we are opening this to UI, do you think we shall check the date is not too loooong ago? Like checking the date is within a year for example?

Reply:
Or is it already check somewhere else?

Reply:
I don't think it's a concern just yet as we are doing all of it right now.

Reply:
Exactly, right. Its the correct question to ask, but they can put 10000 BC if they want, this give them the option to have as much or as little backfill as they choose(for commits).